Implemented searching

stevenpi opened the pull request:
According to the issue, `Haystack` was integrated in the project, with the `Whoosh` backend

mbaechtold commented on Blog/templates/search/search.html:
```diff
@@ -0,0 +1,47 @@
+                    </p>
+                </div>
+            {% empty %}
+                <p>No results found.</p>
```
Shouldn't this be translated too?

stevenpi commented:
I removed the files in the `whoosh_index` folder from github and ignored the folder by logging it in the `.gitignore` file  
Described how to rebuild the searchindex in `Readme.md`
Added the last translation
